### PR TITLE
Style check

### PR DESCRIPTION
**Pull request by ChristinaXu2017**
# Description

Polishing code based on google style check suggestion includes spaces around brackets;  line change; variable name pattern; libraries import order etc.  Updating google style sheet, now it allows a maximum of four continuing uppercase letters in a variable name, such as germelineNNS; Before it only allows one uppercase. This rule also apply to final variable.  All changes  in this branch only happen on q3indel.

## Type of change

Please delete options that are not relevant.

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

use existing unit tests
 

# Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: q3indel/src/au/edu/qimr/indel/pileup/ReadIndels.java [hunk 1]
```diff
@@ -257,13 +269,30 @@ public void resetGenotype(VcfRecord vcf){
 		return indelPositionMap;
 	}
 	
-	public VcfHeader getVcfHeader(){ return header;	}
+	public VcfHeader getVcfHeader() { 
+		return header;	
+	}
+	
+	int getCounts_newIndel() {
```
Comment (holmeso) [inline q3indel/src/au/edu/qimr/indel/pileup/ReadIndels.java:276]: remove underscore

File: q3indel/src/au/edu/qimr/indel/pileup/ReadIndels.java [hunk 2]
```diff
@@ -257,13 +269,30 @@ public void resetGenotype(VcfRecord vcf){
...
-	public VcfHeader getVcfHeader(){ return header;	}
+	public VcfHeader getVcfHeader() { 
+		return header;	
+	}
+	
+	int getCounts_newIndel() { 
+		return counts[0];
+	}
+	
+	int getCounts_overlapIndel() {
```
Comment (holmeso) [inline q3indel/src/au/edu/qimr/indel/pileup/ReadIndels.java:280]: remove underscore

File: q3indel/src/au/edu/qimr/indel/pileup/ReadIndels.java [hunk 3]
```diff
@@ -257,13 +269,30 @@ public void resetGenotype(VcfRecord vcf){
...
+	}
 	
-	int getCounts_newIndel(){ return counts[0]; }
-	int getCounts_overlapIndel(){ return counts[1]; }
-	int getCounts_multiIndel(){ return counts[2]; }
-	int getCounts_inputLine(){ return counts[3]; }	
-	int getCounts_inputMultiAlt(){ return counts[4]; }
-	int getCounts_totalIndel(){return positionRecordMap.size();}
+	int getCounts_inputLine() { 
+		return counts[3];
```
Comment (holmeso) [inline q3indel/src/au/edu/qimr/indel/pileup/ReadIndels.java:288]: remove underscore

File: q3indel/src/au/edu/qimr/indel/pileup/ReadIndels.java [hunk 4]
```diff
@@ -257,13 +269,30 @@ public void resetGenotype(VcfRecord vcf){
...
-	int getCounts_overlapIndel(){ return counts[1]; }
-	int getCounts_multiIndel(){ return counts[2]; }
-	int getCounts_inputLine(){ return counts[3]; }	
-	int getCounts_inputMultiAlt(){ return counts[4]; }
-	int getCounts_totalIndel(){return positionRecordMap.size();}
+	int getCounts_inputLine() { 
+		return counts[3]; 
+	}	
 	
+	int getCounts_inputMultiAlt() {
```
Comment (holmeso) [inline q3indel/src/au/edu/qimr/indel/pileup/ReadIndels.java:291]: remove underscore

File: q3indel/src/au/edu/qimr/indel/pileup/ReadIndels.java [hunk 5]
```diff
@@ -257,13 +269,30 @@ public void resetGenotype(VcfRecord vcf){
...
-	int getCounts_totalIndel(){return positionRecordMap.size();}
+	int getCounts_inputLine() { 
+		return counts[3]; 
+	}	
 	
+	int getCounts_inputMultiAlt() { 
+		return counts[4];
+	}
+	
+	int getCounts_totalIndel() {
```
Comment (holmeso) [inline q3indel/src/au/edu/qimr/indel/pileup/ReadIndels.java:295]: remove underscore

File: q3indel/src/au/edu/qimr/indel/pileup/ReadIndels.java (hunk 1 above)
Comment (ChristinaXu2017) [reply q3indel/src/au/edu/qimr/indel/pileup/ReadIndels.java:276]: done

File: q3indel/src/au/edu/qimr/indel/pileup/ReadIndels.java (hunk 2 above)
Comment (ChristinaXu2017) [reply q3indel/src/au/edu/qimr/indel/pileup/ReadIndels.java:280]: done

File: q3indel/src/au/edu/qimr/indel/pileup/ReadIndels.java (hunk 3 above)
Comment (ChristinaXu2017) [reply q3indel/src/au/edu/qimr/indel/pileup/ReadIndels.java:288]: done

File: q3indel/src/au/edu/qimr/indel/pileup/ReadIndels.java (hunk 4 above)
Comment (ChristinaXu2017) [reply q3indel/src/au/edu/qimr/indel/pileup/ReadIndels.java:291]: done

File: q3indel/src/au/edu/qimr/indel/pileup/ReadIndels.java (hunk 5 above)
Comment (ChristinaXu2017) [reply q3indel/src/au/edu/qimr/indel/pileup/ReadIndels.java:295]: done

**Pull request merged**